crates/common/certificate/src/parse_root_certificate.rs

```diff
@@ -21,14 +21,31 @@ pub fn create_tls_config(
 
 fn new_root_store(cert_path: PathBuf) -> Result<RootCertStore, CertificateError> {
     let mut root_store = RootCertStore::empty();
+    rec_add_root_cert(&mut root_store, cert_path);
+    Ok(root_store)
+}
+
+fn rec_add_root_cert(root_store: &mut RootCertStore, cert_path: PathBuf) {
+    if let Err(err) = try_rec_add_root_cert(root_store, cert_path.clone()) {
+        eprintln!("Ignoring any certificate in {:?} : {}", cert_path, err)
```

Comment:
Difficult to understand this error message.

Reply:
I propose:

```
            "Ignoring certificates in {:?} due to {}",
```

Reply:
ok